client/test/alice.go

```diff
@@ -46,8 +46,6 @@ func (r *Alice) Execute(cfg ExecConfig) {
 func (r *Alice) exec(_cfg ExecConfig, ch *paymentChannel) {
 	cfg := _cfg.(*AliceBobExecConfig)
 	we, them := r.Idxs(cfg.Peers())
-	// 1st stage - channel controller set up
-	r.waitStage()
```

Comment:
Line 38 probably needs to be changed from `4` to `3`.

Reply:
That's right, good catch!